app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingScreen.kt

```diff
@@ -673,6 +724,33 @@ private fun DownloadBitwardenActionCard(
     },
 )
 
+@Composable
+private fun SyncWithBitwardenActionCard(
+    modifier: Modifier = Modifier,
+    onDismissClick: () -> Unit,
+    onSyncWithBitwardenClick: () -> Unit,
+) = BitwardenActionCard(
+    modifier = modifier,
+    actionIcon = rememberVectorPainter(R.drawable.ic_bitwarden), // TODO: update this icon
```

Comment:
Is this `TODO` associated with another ticket?

Reply:
🤦 thanks or noticing this. This was a note for myself to 

I'm gonna start using `//FIXME` for these so detekt fails.

Updated: 5181f0aeaf61f1e11a72270112b74132bfdd3f7f
![Screenshot_1729537825](https://github.com/user-attachments/assets/e93818af-2b89-4db3-811d-d65fa633a89b)

Reply:
Ha. Happens to me all the time too. `//FIXME` is a good idea since detekt looks for it. I tried `//STOP-SHIP` a long time ago, but that causes AS to fail compilation. Or at least it used to. 😢